MSAL/MSAL.xcodeproj/project.pbxproj

```diff
@@ -8011,12 +8010,13 @@
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				CODE_SIGN_ENTITLEMENTS = test/app/mac/MSALMacTestApp.entitlements;
-				CODE_SIGN_IDENTITY = "Mac Developer";
+				CODE_SIGN_IDENTITY = "Apple Development";
```

Comment:
Just curious what is this change for? I think I always use Automatically manage signing to let Xcode decide a profile when building test app

Reply:
I do not have this `Mac Developer` anymore and it is default to `Apple Development`. 

This is still Automatically managed and there is no change on this part